plugins/sharding/logical_range_filter.rb

```diff
@@ -1112,6 +1118,13 @@ def sort_result_set(result_set)
           if @context.current_limit > 0
             @context.current_limit -= sorted_result_set.size
           end
+          key_order_pairs = ""
+          sort_keys.each do |pair|
+            key_order_pairs << "key: #{pair[:key]} order: #{pair[:order]}"
+          end
```

Comment:
`KEY` for asc and `-KEY` for desc is better. We use it in other places.

Reply:
Thanks for review.